res/logback.xml

```diff
@@ -0,0 +1,26 @@
+<configuration>
+    <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>briefcase.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>briefcase.%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+            <!-- keep 14 days' worth of history capped at 1GB total size -->
```

Comment:
Looks like it's actually 10 days for a total of 100MB?

Reply:
Ah, I put 14 days first and then I changed it to 10. Will update 👯‍♂️ .